helm/nostr/strfry-chart/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for strfry-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: dockurr/strfry
+  tag: latest
+  pullPolicy: IfNotPresent
+
+persistence:
+  enabled: true
+  size: 1Gi
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+strfry-chart:
```

Comment:
no change required here, but these values are unnecessary since they seem to be the same (or similar) to the default values,